.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,126 @@
+---
+name: Bug Report
+description: Report a problem with dlt
+labels: ["kind:bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for reporting a bug for dlt! Please fill out the sections below.
+        If you are not sure if this is a bug or not, please join our [Slack](https://join.slack.com/t/dlthub-community/shared_invite/zt-1slox199h-HAE7EQoXmstkP_bTqal65g)
+        and ask in the #3-technical-help channel.
+  - type: dropdown
```

Comment:
should be it a normal input? so people just paste the version

Reply:
Good point. Let's change that, also reduces the maintenance